src/materials/czm/CZMMaterialBasePK1.C

```diff
@@ -199,6 +232,10 @@ void CZMMaterialBasePK1::initKinematicsVariale() {
   _C = _DR_avg[_qp] * _Q0.transpose();
   _D = _R_avg[_qp] * _Q0.transpose();
   _B = _dadot_da_avg[_qp] * _D;
+
```

Comment:
Similar, these seem to happen in opt.

Reply:
no this is correct, mooseAssert get compiled only in devel and dbg mode